include/boost/url/authority_view.hpp

```diff
@@ -1224,7 +1224,7 @@ BOOST_URL_DECL
 std::ostream&
 operator<<(
     std::ostream& os,
-    authority_view const& a);
+    authority_view const& u);
```

Comment:
What's the point of renaming this? I was going to use `a` as the canonical variable name for authority views.

Reply:
The var name wasn't matching the documentation. This generates a doxygen warning. So I'll change the documentation instead.